inst/rmarkdown/templates/frontiers_article/skeleton/frontiersSCNS.cls

```diff
@@ -1,955 +1,955 @@
-\newcommand\classname{frontiersSCNS} %this file is based in bioninfo.cls

```

Comment:
Are these `CRLF <-> LF` changes?

Reply:
Hmm - I edited in texMaker, so it may have done that without me realizing.

Reply:
Should only be an edit to `epstopdf` package exclusion and `logo.pdf`